README.md

```diff
@@ -2,11 +2,30 @@
   <img height=75 src="https://github.com/smartface/sample-clienteling/blob/master/temp/smartface_logo.png" alt="smartface Logo" />
 </h3>
 
-# Clienteling from Smartface
+# Sample Customer Targeting Application from Smartface
 [![Twitter: @Smartface_io](https://img.shields.io/badge/contact-@Smartface_io-blue.svg?style=flat)](https://twitter.com/smartface_io)
 [![License](https://img.shields.io/badge/license-MIT-green.svg?style=flat)](https://github.com/smartface/sample-clienteling/blob/master/LICENSE.md)
 
-A sample app to demonstrate Clienteling Application that can be developed by Smartface.
+Sample Customer Targeting Project
+
+<div style="display:flex;">
+<img width=250 src="https://github.com/omerbulgak/sample-clienteling/blob/master/temp/ct1.jpeg">
+<img width=250 src="https://github.com/omerbulgak/sample-clienteling/blob/master/temp/ct2.jpeg" hspace="30">
+  <img width=250 src="https://github.com/omerbulgak/sample-clienteling/blob/master/temp/ct3.jpeg">
+  <img width=250 src="https://github.com/omerbulgak/sample-clienteling/blob/master/temp/ct4.jpeg">
+  <img width=250 src="https://github.com/omerbulgak/sample-clienteling/blob/master/temp/ct5.jpeg" hspace="30">
+  <img width=250 src="https://github.com/omerbulgak/sample-clienteling/blob/master/temp/ct6.jpeg">
+</div>
+
+## Getting the Project
```

Comment:
Instead of explaining how, a reference could be more useful. https://developer.smartface.io/docs/create-new-project